libs/platform/src/lib/form/form-group/form-field/form-field.component.ts

```diff
@@ -94,7 +96,7 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
      * Form field id
      */
     @Input()
-    id: string;
+    id: string = (++defaultId).toString();
```

Comment:
having id=1 (example) is very fragile on my opinion, could we extend it?

Reply:
oh, i see, it gets extended in the template. but value of this field used as a suffix, could we mention it in the typedoc?

Reply:
actually it felt more reasonable to use the whole id than a suffix, updated